src/main/java/com/conveyal/r5/profile/PathWithTimes.java

```diff
@@ -218,9 +225,22 @@ public int compareTo(Itinerary o) {
         }
     }
 
+    public int hashCode () {
+        return super.hashCode() ^ (accessMode.hashCode() * 31 + egressMode.hashCode() * 29);
+    }
+
+    public boolean equals (Object o) {
```

Comment:
Defining hash code and equals functions that do not compare / include all fields requires very clear documentation as to why this is the case, and how it is used. It appears to also be the case for the superclass.